src/config.rs

```diff
@@ -15,6 +15,13 @@ pub fn default_config_path(mut home_dir: PathBuf) -> String {
     home_dir.to_str().unwrap().to_owned()
 }
 
+pub fn global_config_path() -> String {
+    let mut path = PathBuf::from("/etc/puppetlabs/client-tools");
+    path.push("puppetdb");
+    path.set_extension("conf");
+    path.to_str().unwrap().to_owned()
```

Comment:
So it looks like what's going on here is path.to_str() is creating an &str of the path. Then the to_owned() call is copying the string reference to a String and giving ownership to the caller?